Sources/NDT7Settings.swift

```diff
@@ -113,110 +80,79 @@ public struct NDT7Timeouts {
     }
 }
 
-/// Mlab NDT7 Server.
-public struct NDT7Server: Codable {
-
-    /// Last server got from MLab
-    static var lastServer: NDT7Server?
-
-    /// ip array
-    public var ip: [String]?
-
-    /// country
-    public var country: String?
-
-    /// city
-    public var city: String?
+public struct LocateAPIV2Response: Codable {
+    public var results: [NDT7ServerV2]
+}
 
-    /// fqdn
-    public var fqdn: String?
+/// Response type for the new, Locate API V2 response
+public struct NDT7ServerV2: Codable {
+    public var machine: String
+    
+    public var location: NDT7Location
+    
+    public var urls: NDT7LocationUrls
+}
 
-    /// site
-    public var site: String?
+public struct NDT7Location: Codable {
+    public var city: String
+    
+    public var country: String
 }
 
-/// This extension provides helper methods to discover Mlab servers availables.
-extension NDT7Server {
+/// Upload and download URLs for the V2 Location API
+public struct NDT7LocationUrls: Codable {
+    public var downloadUrl: String
+    
+    public var uploadUrl: String
+    
+    public var insecureDownloadUrl: String
```

Comment:
Please, specify comments about the parameters, to know for example what is the difference between download and insecure download.